examples/using-redux-w-interaction-code-splitting/gatsby-browser.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Implement Gatsby's Browser APIs in this file.
+ *
+ * See: https://www.gatsbyjs.com/docs/browser-apis/
+ */
+
+// You can delete this file if you're not using it
+
+export { default as wrapRootElement } from "./src/redux/reduxWrapper"
```

Comment:
To have proper hot reloading you'll want to call the `wrapRootElement` api and use a component that is in PascalCase (Fast Refresh doesn't like lowercased components).

See https://github.com/system-ui/theme-ui/blob/develop/examples/gatsby/gatsby-browser.js for an example